src/Domain/Person.php

```diff
@@ -212,22 +212,25 @@ class Person
      * @var string|null From residential address, if donor is claiming Gift Aid.
      */
     #[ORM\Column(type: 'string', nullable: true)]
+    #[Assert\Length(max: 255, groups: [self::VALIDATION_COMPLETE, self::VALIDATION_NEW])]
     public ?string $home_address_line_1 = null;
 
     /**
      * @OA\Property()
      * @var string|null From residential address, if donor is claiming Gift Aid and is GB-resident.
      */
     #[ORM\Column(type: 'string', nullable: true)]
+    #[Assert\Length(max: 10, groups: [self::VALIDATION_COMPLETE, self::VALIDATION_NEW])]
     public ?string $home_postcode = null;
 
     /**
      * @OA\Property()
-     * @var string|null From residential address, if donor is claiming Gift Aid. Can be 'GB' or 'OVERSEAS',
-     *                  or null if not applicable. Consuming code should assume that additional ISO 3166-1
-     *                  alpha-2 country codes could be set in the future.
+     * @var string|null From residential address, if donor is claiming Gift Aid, or set in account page. Can be 'GB', or
+     * any other two letter country code, or 'OVERSEAS' if specific non UK country is not known. or null if not
```

Comment:
I think perhaps useful to keep the reference to ISO 3166-1 to make it easy to look up the relevant list? Not a blocker.

Reply:
Thanks, done in 1fa6fba